### PR TITLE
footer navigation

### PR DESCRIPTION
**Pull request by Srishtiii19**
Screenshot (366)

### CODE REVIEW

**Comment by Srishtiii19** [comment]
i have made footer @pooranjoyb, if any changes required then please let me know i'll update it.

**Comment by pooranjoyb** [comment]
Seems like a complete new design. Please don't change the overall design pattern. 
I guess you were only supposed to add the links and work on the navigation, right? @Srishtiii19

**Comment by Srishtiii19** [comment]
hello @pooranjoyb, there seem to be some confusion from my end, i'll change the bar. Also can you please mention icons you would want in the bar?

**Comment by pooranjoyb** [comment]
> hello @pooranjoyb, there seem to be some confusion from my end, i'll change the bar. Also can you please mention icons you would want in the bar?

Yes, I mean, just do the routings, you don't need to change the Footer design or anything :)

**Comment by Srishtiii19** [comment]
Screenshot (369)

**Comment by Srishtiii19** [comment]
ohhh, the footer is already is made, i have seen that, i just have to link add links to it for navigation, right?

**Comment by pooranjoyb** [comment]
> ohhh, the footer is already is made, i have seen that, i just have to link add links to it for navigation, right?

Haanji :) 
Just the navagations using `Link` of `react-router-dom`
@Srishtiii19

File: index.html [hunk 1]
```diff
@@ -6,8 +6,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>PopShop</title>
     <link rel="stylesheet" href="/src/tailwind/output.css">
+    <script src="https://cdn.tailwindcss.com"></script>
```
Comment (pooranjoyb) [inline index.html:9]: remove this, follow contribution guidelines, you need to run `npm run watch-css` to generate tailiwind-css.

File: src/pages/Error.tsx [hunk 2]
```diff
@@ -19,7 +19,107 @@ function Error() {
       </div>
 
       {/* Footer  */}
+
       <Footer />
+     
+      <footer className="text-gray-600 body-font">
```
Comment (pooranjoyb) [inline src/pages/Error.tsx:25]: Remove this entire Footer, udpate the `Footer` Component

**Review by pooranjoyb — CHANGES_REQUESTED** [review]
@Mahitej28 @Srishtiii19

File: src/pages/Error.tsx (hunk 2 above)
Comment (Srishtiii19) [reply src/pages/Error.tsx:25]: yes sir doing it, removed the section that i have added now updating navigation in the footer via links @pooranjoyb

**Comment by Srishtiii19** [comment]
hello, @pooranjoyb @Mahitej28 i have made changes, please check

**Review by Mahitej28 — CHANGES_REQUESTED** [review]
Dont change the links that cant be redirected.

**Comment by Mahitej28** [comment]
@Srishtiii19 do the changes and also i advice u take a pull again from master and dev branch

**Comment by Srishtiii19** [comment]
work done as said @Mahitej28

File: src/components/Footer.tsx [hunk 3]
```diff
@@ -8,167 +8,191 @@ import { FaDribbble } from "react-icons/fa";
...
+                      to="/"
+                      className="text-gray-700 transition hover:opacity-75"
+                    >
+                      {" "}
+                      Profile{" "}
+                    </Link>
+                  </li>
+                  <li>
+                    <Link
+                      to="/cart"
```
Comment (Mahitej28) [inline src/components/Footer.tsx:124]: @Srishtiii19 the link should be "/home/cart" just double-check all the links where you are routing. Kindly review all the links again.

**Review by Mahitej28 — CHANGES_REQUESTED** [review]
Kindly review all the links again and do the requested changes @Srishtiii19

File: src/components/Footer.tsx (hunk 3 above)
Comment (Srishtiii19) [reply src/components/Footer.tsx:124]: sure, will do it.

**Comment by Mahitej28** [comment]
@Srishtiii19 updates?

**Comment by Srishtiii19** [comment]
> @Srishtiii19 updates?

Hello @Mahitej28
I am at college rn, will complete this work today itself.

**Comment by Srishtiii19** [comment]
Screenshot (373)

**Comment by Srishtiii19** [comment]
i dont know why it is showing error, wasnt showing before. Working on it.

**Comment by pooranjoyb** [comment]
> i dont know why it is showing error, wasnt showing before. Working on it.

Run `npm install react-icons`

**Comment by Srishtiii19** [comment]
Screenshot (375)

Unable to open any other page, please help

**Comment by pooranjoyb** [comment]
did you follow the `contributing guidelines`?
Cross check everything with it. @Srishtiii19

**Comment by pooranjoyb** [comment]
> > did you follow the `contributing guidelines`? Cross check everything with it. @Srishtiii19
> 
> hello @pooranjoyb, still facing the same issue none of the pages are opening except the login page. This wasn't an issue before.

DM in discord.

**Comment by Mahitej28** [comment]
@Srishtiii19 any updates? if the issue is still not solved yet DM me on Discord.

**Comment by Mahitej28** [comment]
Completed

**Pull request closed without merging**